plugin/src/sbt-test/scalafmt-sbt/sbt/build.sbt

```diff
@@ -5,6 +5,16 @@ ThisBuild / fork := true
 
 lazy val p123 = project
   .in(file("."))
+  .settings(
+    InputKey[Unit]("updateLastModified") := {
+      import sbt.internal.util.complete.DefaultParsers._
+      import scala.sys.process.Process
+      val f = spaceDelimited("").parsed.head
+      Process(Seq("chmod", "666", f)).!
```

Comment:
why do we need chmod? why not let touch fail?

Reply:
latest sbt 1.x and 2.x `touch` fail if permission is `000`.

Reply:
is that the only reason to introduce this new method `updateLastModified`, instead of `touch`? if so, does it need to succeed in such cases? i'd prefer to have it fail, and to use chmod explicitly in those test cases when we want to touch a non-writable file.

Reply:
this does look more intentional to me, I'd prefer it if not too much trouble.